app/views/csv_exports/index.html.erb

```diff
@@ -1,8 +1,9 @@
 <h1>Here are your active reports</h1>
 
 <section id="csv-exports" class="clearfix">
-  <div class="deploy-csv">
-    <%= link_to "Start a CSV export job", new_csv_export_path %>
+  <div>
+    <p><%= link_to "Create Deploys Report", new_csv_export_path %></p>
```

Comment:
could remove the `Create` part since hitting that link does not create it ... `New Deploys Report` or just `Deploys Report`